jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentials.java

```diff
@@ -108,4 +110,20 @@ public String getPasswordDescriptor() {
           }
         });
   }
+
+  @Nullable
+  @VisibleForTesting
+  Server getServerFromMavenSettings(String registry) {
+    // Find and remove port (if present)
```

Comment:
Oh, this comment is now confusing. Let's update and move this to line 123:

```java
// try without port
int index = registry.lastIndexOf(':');
```

(Sorry, I should have caught this earily.)

Reply:
Never mind. I just did it myself.